### ISSUE TITLE
Required fields validation on nested objects

### ISSUE THREAD

**Issue by gersmann** [bhch/django-jsonform#96]
Hi @bhch, thank you for supporting the 'required' attribute. I tested it on a schema, and it is still a bit spotty. 

For instance:
```
{
   "properties":{
      "object1":{
         "type":"object",
         "properties":{
            "prop1":{
               "type":"string"
            },
            "prop2":{
               "type":"string"
            }
         },
         "required":[
            "prop1",
            "prop2"
         ],
         "additionalProperties":false
      },
      "object2":{
         "type":"object",
         "properties":{
            "prop3":{
               "type":"string"
            },
            "object3":{
               "type":"object",
               "properties":{
                  "prop4":{
                     "type":"string"
                  },
                  "prop5":{
                     "type":"string"
                  }
               },
               "required":[
                  "prop1",
                  "prop2"
               ],
               "additionalProperties":false
            }
         },
         "required":[
            "prop3",
            "object3"
         ],
         "additionalProperties":false
      }
   },
   "required":[
      "object1"
   ],
   "additionalProperties":false
}
```

This only marks prop3 as required, even though all properties are, but object2 is not.

**Comment by bhch** [comment]
Ah, yes, I've found the cause of this issue. I'll fix it soon.

**Comment by bhch** [comment]
Fix released in [v2.16.1](https://github.com/bhch/django-jsonform/releases/tag/v2.16.1). Please update. Thanks for reporting this bug.

---

P.S.: The schema that you posted needs a minor correction. In `object3` the `required` list should be `["prop4", "prop5"]` (not `["prop1", "prop2"]`).
